backend/app/Listeners/NotifyUsersAboutUpdatedSubmissionStatus.php

```diff
@@ -112,7 +132,7 @@ public function acceptedForReview($default)
     {
         $default['subject'] = 'Submission Accepted for Review';
         $default['body'] = $default['submission']['title'] . ' has been accepted for review. ' .
-                           'It is currently awaiting review from the assigned reviewers.';
+                           'Once it is opened for review, it will be available for review.';
```

Comment:
Just a note that we should create a ticket for translating these strings and/or making the email templates editable by administrators

Reply:
I say create a ticket because I don't think how we do this will be straightforward

Reply:
100% agree. I'd like to be able to implement this at some point: https://laravel.com/docs/9.x/localization

Reply:
The issue has been created: https://app.zenhub.com/workspaces/ccr-development-5ff39ba363391e0019713c87/issues/mesh-research/ccr/1465